update_issue_attributes.py

```diff
@@ -0,0 +1,96 @@
+"""
+Delete purchases.
+Deletes all PO lines (ones associated with aboms, kits, or that have been received) can't be deleted. 
+Deletes all purchase orders that don't have receipts or PO lines, approvals, fees, or approval requests.
+You can additionally add PO ids that you don't want deleted in PURCHASES_TO_SKIP
+"""
+import os
+import sys
+import inspect
+import csv
+
+# Reset the path so it can be run from the parent directory
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir)
+
+import argparse
+from utilities.api import Api
+import queries
+from config import config
+
+import logging
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)-8s %(message)s",
+    datefmt="%a, %d %b %Y %H:%M:%S",
+    filename="delete_purchases/log.txt",
+    filemode="w",
+)
+
+def get_issues(api):
+    request_body = {"query": queries.GET_ISSUE_ATTRIBUTES}
+    return api.request(request_body)["data"]
+
+def update_issue_attribute(issues,api):
+    file = open("issues.csv")
+    csvreader = csv.reader(file)
+
+    rows = []
+    for row in csvreader:
+        rows.append(row)
+    file.close()
+
+    
+    for issue in issues["issues"]["edges"]:
+        issue_id = issue["node"]["id"]
+        for attribute_etag in issue["node"]["attributes"]:
+            etag = attribute_etag["Etag"]
+        for row in rows:
+            if issue_id == int(row[0]):
+                value = row[1]
+            continue
+
+    
+        request_body = {
+            "query": queries.UPDATE_ISSUE_ATTRIBUTE,
+            "variables": {
+                        "input": {
+                            "issueId": issue_id,
+                            "key": "Defect Code",
+                            "value": value,
+                            "etag": etag
+                        }
+            }
+        }
+        api.request(request_body)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Bulk print location barcode labels.")
```

Comment:
Barcode description in argument parser

Reply:
I think I made this mistake in the PO one too

Reply:
@mattemerzian hmm, looks like we never use that variable anyway, in either script? Can that line be removed?